packages/serve/test/auth/cannerPATAuthenticator.spec.ts

```diff
@@ -0,0 +1,237 @@
+import * as sinon from 'ts-sinon';
+import { IncomingHttpHeaders } from 'http';
+import { Request, BaseResponse } from 'koa';
+import { CannerPATAuthenticator } from '@vulcan-sql/serve/auth';
+import { AuthResult, AuthStatus, KoaContext } from '@vulcan-sql/serve/models';
+
+const wrappedAuthCredential = async (
```

Comment:
Wrap the function to reuse it great!

However, according to your test cases for asserting the error,  you should prevent including other non-target test logistics, like here you create the authenticator and called the activate method, it is not the target act code section even though these create heavier and activate behavior seems not possibly cause others errors.